src/views/pages/Login/index.vue

```diff
@@ -27,7 +27,7 @@ export default Vue.extend({
   methods: {
     async login() {
       console.log('attempting login');
-      await axios.post('https://boresha.live:19443/actions/authenticate', {
+      await axios.post(`${process.env.CORE_HOST}:${process.env.CORE_PORT}/actions/authenticate`, {
```

Comment:
so are we still using axios? or are we going to use your new changes in another pr?

Reply:
We will be doing everything via vuex at some point. I just did the stuff to do the data sync, login still has to be done